src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1049,25 +1049,22 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
 
                     if (body != null)
                     {
-                        (SyntaxNode Syntax, BoundNode Body, ExecutableCodeBinder Binder, NullableWalker.SnapshotManager SnapshotManager) forSemanticModelToUseInLambda = forSemanticModel;
+                        MethodBodySemanticModel.InitialState forSemanticModelToUseInLambda = forSemanticModel;
```

Comment:
>forSemanticModelToUseInLambda [](start = 61, length = 29)

Is this local needed, or could we use `forSemanticModel` directly?

Reply:
I _think_ we could use `forSemanticModel` directly, but I didn't want to change it just in case this was doing something I didn't understand with capturing.

---
In reply to: [305051587](https://github.com/dotnet/roslyn/pull/37322#discussion_r305051587) [](ancestors = 305051587)